app/views/provider/admin/webhooks/edit.html.slim

```diff
@@ -1,13 +1,75 @@
-- content_for :javascripts, javascript_include_tag('provider/admin/webhooks')
-- content_for(:title)
-  = "Webhooks"
-- content_for :page_header_title, 'Webhooks'
-- content_for :page_header_body do
-  ' Webhooks let you define a URL to be called with a notification when events in the 3scale
-  ' management system happen.
-
-= semantic_form_for @webhook, url: provider_admin_webhooks_path do |f|
-  = render 'form', form: f
-
-  = f.actions do
-    = f.commit_button I18n.t('provider.admin.webhooks.edit.form.submit_button_label')
+- content_for :title, t('.header.title')
+- content_for :page_header_title, t('.header.title')
+- content_for :page_header_body, t('.header.body')
+
+- content_for :javascripts do
+  = javascript_packs_with_chunks_tag 'webhooks_edit'
+
+div class="pf-c-card"
+  div class="pf-c-card__body"
+    = semantic_form_for @webhook, url: provider_admin_webhooks_path,
+                                  builder: Fields::PatternflyFormBuilder,
+                                  html: { class: 'pf-c-form pf-m-limit-width' } do |form|
+      section class="pf-c-form__section" role="group"
+        div class="pf-c-form__section-title" aria-hidden="true" Behavior
+        div class="pf-c-form__group"
+          = form.input :active, as: :patternfly_checkbox
+          = form.input :provider_actions, as: :patternfly_checkbox
+
+      section class="pf-c-form__section" role="group"
+        div class="pf-c-form__section-title" aria-hidden="true" Webhooks endpoint
+        div class="pf-c-form__group"
+          - show_action = @webhook.persisted? && @webhook.url.present?
+          = form.input :url, as: :patternfly_input, action:(show_action ? { title: 'Ping!', 'data-ping-url': provider_admin_webhooks_path } : nil)
```

Comment:
It's kinda weird, that you need to actually enable webhooks, and set the URL, **and** save the form, and only then the Ping button will appear. But that was the previous behavior also, so it's fine.